src/ipf.cpp

```diff
@@ -351,7 +351,7 @@ static void ReadTrack (t_track *pt_)
   if (!pt_->data)
   {
     // Allocate enough for the full decoded size, allowing for expanded overlapping sectors
-    memcpy(pt_->data = static_cast<byte*>(malloc(uDecoded)), abDecoded, pt_->size = uDecoded);
+    memcpy(pt_->data = new byte[uDecoded], abDecoded, pt_->size = uDecoded);
```

Comment:
It seems we're never freeing this ... Not a big deal as we're not reallocating over and over but could maybe be improved (e.g unique_ptr or shared_ptr). I'm adding a TODO

Reply:
Yes it never does indeed :)